internal/pkg/agent/application/actions/handlers/handler_action_policy_change.go

```diff
@@ -179,6 +179,12 @@ func testFleetConfig(ctx context.Context, log *logger.Logger, clientConfig remot
 			errors.TypeNetwork, errors.M("hosts", clientConfig.Hosts))
 	}
 
+	if resp.StatusCode != http.StatusOK {
+		return errors.New(
```

Comment:
Can we add a test for this? Seems like we already have the httptest server setup to do it.

Reply:
Done in 426c30ea6aacf7886446dbbe51d7ab472a4881bf